server/graphql/purchase_order/src/mutations/insert.rs

```diff
@@ -0,0 +1,71 @@
+use async_graphql::*;
+use graphql_core::standard_graphql_error::StandardGraphqlError::{BadUserInput, InternalError};
+use graphql_core::ContextExt;
+use graphql_types::types::IdResponse;
+use repository::PurchaseOrderRow;
+use service::purchase_order::insert::{
+    InsertPurchaseOrderError as ServiceError, InsertPurchaseOrderInput as ServiceInput,
+};
+
+#[derive(InputObject)]
+#[graphql(name = "InsertPurchaseOrderInput")]
+pub struct InsertInput {
+    pub id: String,
+    pub supplier_name_link_id: String,
```

Comment:
I think we need this to be "supplier_id" not the link ID. Pretty sure front-end has no knowledge of the "linked" id, they'll only no what the actual ID of the supplier is -- is that right?

Reply:
Yes great point, `supplier_id` makes sense